lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -396,6 +402,16 @@
     path: /var/tmp/test_results
     state: directory
     mode: 0777
+  when: preburn
+
+- name: Install multipath-tools
```

Comment:
We had an error in the Ansible role here that was ignored before (and the same with the `gtest`/`gmock` installation step). Both my changes fixes the errors, but it may be that we could remove the steps for good since we don't need these packages.